components/InputList/InputListItem.tsx

```diff
@@ -32,7 +32,7 @@ export const InputListItem: FC<InputListItemProps> = ({
     }
   };
 
-  const handleChange = (e) => {
+  const handleChange: React.ChangeEventHandler<HTMLInputElement> = (e) => {
```

Comment:
Manual change here so that `e` is not implicitly `any`. Now our function as a fully-typed signature.